magellan-jetpack.json

```diff
@@ -1,7 +1,8 @@
 {
   "mocha_tests": [
     "specs",
-    "specs-jetpack-calypso"
+		"specs-jetpack-calypso",
+		"specs-blocks"
```

Comment:
Are these 2 lines intentionally indented?

Reply:
Fixed in 4527fb1